.travis.yml

```diff
@@ -39,13 +39,13 @@ script:
       ( cd service_crategen && \
         cargo run -- -c ./services.json -o ../rusoto/services && \
         cd .. )
-  - echo "Running Rusoto tests" && cd rusoto && cargo test --lib --all -v && cd ..
+  - echo "Running Rusoto tests" && cargo test --lib --all -v
   - |
       echo "Running cargo docs on stable Rust on Linux" &&
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-        cd rusoto && cargo doc --all --no-deps && cd ..
+        cargo doc --all --no-deps
       fi
-after_success:
+      after_success:
```

Comment:
Not sure why my editor didn't see that, it catches most other whitespace. Just fixed.